components/TheFooter.vue

```diff
@@ -119,6 +119,24 @@
                   </clipPath>
                 </defs>
               </svg>
+              <!-- farcaster doesnt have a font awesome icon -->
+              <svg
+                v-else-if="item.icon === 'farcaster'"
+                width="20"
+                height="20"
+                viewBox="0 0 1000 800"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg">
+                <path
+                  d="M257.778 155.556H742.222V844.445H671.111V528.889H670.414C662.554 441.677 589.258 373.333 500 373.333C410.742 373.333 337.446 441.677 329.586 528.889H328.889V844.445H257.778V155.556Z"
+                  fill="white" />
+                <path
+                  d="M128.889 253.333L157.778 351.111H182.222V746.667C169.949 746.667 160 756.616 160 768.889V795.556H155.556C143.283 795.556 133.333 805.505 133.333 817.778V844.445H382.222V817.778C382.222 805.505 372.273 795.556 360 795.556H355.556V768.889C355.556 756.616 345.606 746.667 333.333 746.667H306.667V253.333H128.889Z"
+                  fill="white" />
```

Comment:
Could you pls move the inline-svg to another path as a single file?
(public/icons/xxx.svg)

Reply:
updated as you suggested